packages/ozone/src/daemon/scheduled-action-processor.ts

```diff
@@ -142,12 +158,14 @@ export class ScheduledActionProcessor {
   }
 
   async performTakedown({
+    email,
     action,
     event,
     modTool,
     moderationTxn,
     settingService,
   }: {
+    email: { subject: string; content: string }
```

Comment:
You could change this to:
```suggestion
    email: { subject: string; content: string } | null
```

And then in the earlier part of the code only set email if there's content, that'd remove the duplicate "do we have content" check on line 213. Extracting `{ subject: string; content: string }` into a type would also help with ensuring the `email` in `executeScheduledAction` aligns with the expected type here.

Another option is:
```suggestion
    email?: { subject: string; content: string }
```